### PR TITLE
{math}[GCC/13.2.0] tdlib v0.9.3

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/t/tdlib/tdlib-0.9.3-GCC-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+sources = ['%(version)s.tar.gz']
+checksums = ['d1730c98f41dcb23bbd0bd8de9dbec51df015304f28a38935848925901594ae8']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('Boost', '1.83.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
```
Comment (boegel) [inline easybuild/easyconfigs/t/tdlib/tdlib-0.9.3-GCC-13.2.0.eb:21]: @PetrKralCZ How is Python a runtime dependency here exactly?

Comment (PetrKralCZ) [reply easybuild/easyconfigs/t/tdlib/tdlib-0.9.3-GCC-13.2.0.eb:18]: This should only be a build dependency. Fixed in [31ce7fc](https://github.com/easybuilders/easybuild-easyconfigs/pull/21305/commits/31ce7fc97a82a6d1960d684fb0052e7fe5cdb218).

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3123.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/d996c88675f21ac24396733edaf4a9b5 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**